### DISCUSSION TITLE
add_git_tag shouldn't assume you want the lane name in the tag

### DISCUSSION THREAD

**Discussion by simonbromberg** [fastlane/fastlane#17637] [Ideas] [answered]
[add_git_tag](https://docs.fastlane.tools/actions/add_git_tag/) lets you customize the tag grouping, the prefix, the posfix… But it doesn't let you opt out of including the current lane in the tag name (other than just providing a totally custom tag).

Seems reasonable it should be at least a parameter so you can turn it off. Is there a particular reason why it's not configurable?

**Comment by rogerluan** [comment, marked as answer]
Hey @simonbromberg 👋 

At the moment the tag that action builds is the following:

```ruby
tag = "#{options[:grouping]}/#{lane_name}/#{options[:prefix]}#{options[:build_number]}#{options[:postfix]}"
```

As you mentioned, `add_git_tag` allows you to provide a custom tag.

So instead of:

```ruby
add_git_tag(
  grouping: grouping,
  prefix: prefix,
  postfix: postfix,
  build_number: build_number
)
```

You could call:

```ruby
add_git_tag(
  tag: "#{grouping}/#{prefix}#{build_number}#{postfix}"
)
```

Does this help?

**Reply by simonbromberg** [reply]
Yes, thank you know that you can fully customize the tag like that, but it's more convenient/cleaner to use the other form of the method. I was suggesting that there should be a parameter that controls whether the lane name appears in the tag name. Every other part of the tag name in your example is pulled from `options` so why not that one? In many cases I would assume one wouldn't want the lane name in the tag anyway and if you did want it you could always include it via `grouping`.

**Reply by rogerluan** [reply]
Ah I see now @simonbromberg ! Makes sense.

I created this PR to implement that feature for you 🤗 https://github.com/fastlane/fastlane/pull/17792

Be my guest to test it if you want to! Testing steps are included in the PR description 🚀 

Hope that helps you out! 💪
